modules/apps/object/object-service/src/main/java/com/liferay/object/internal/security/permission/util/ObjectEntryPermissionUtil.java

```diff
@@ -0,0 +1,59 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2025 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+package com.liferay.object.internal.security.permission.util;
+
+import com.liferay.account.model.AccountEntry;
+import com.liferay.account.model.AccountEntryOrganizationRel;
+import com.liferay.account.service.AccountEntryOrganizationRelLocalServiceUtil;
+import com.liferay.portal.kernel.exception.PortalException;
+import com.liferay.portal.kernel.model.Organization;
+import com.liferay.portal.kernel.security.permission.PermissionChecker;
+
+/**
+ * @author Carolina Barbosa
+ */
+public class ObjectEntryPermissionUtil {
+
+	public static boolean hasAccountEntryPermission(
+			AccountEntry accountEntry, String actionId, String name,
+			PermissionChecker permissionChecker)
+		throws PortalException {
+
+		if (permissionChecker.hasPermission(
+				accountEntry.getAccountEntryGroupId(), name, 0, actionId)) {
+
+			return true;
+		}
+
+		for (AccountEntryOrganizationRel accountEntryOrganizationRel :
+				AccountEntryOrganizationRelLocalServiceUtil.
```

Comment:
@carolmariaabb lately I'm not seeing usages of the `ServiceUtil` in util classes, the common pattern that I applied and has been accepted is passing the services as parameters. Please pass this service as parameter if it is possible.

Thanks!